JeongSungWon/ch8/ch8_11_1(327p).html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>무작위행보</title>
+    <script>
+        function* randomWalk(c, x0, y0, d){
+            var dx = [1,0,-1,0], dy = [0,1,0,-1];
+            var x = x0;
+            var y = y0;
+            c.strokeStyle = "red";
+            c.globalAlpha=0.25;
+            while(true){
+                yield;
+                c.beginPath();
+                c.moveTo(x,y);
+                var dir = Math.floor(Math.random()*4);
+                x += d*dx[dir];
+                y += d*dy[dir];
+                c.lineTo(x,y);
+                c.stroke();
+            }
+        }
+        window.onload=function(){
+            var canvas = document.getElementById("mycanvas");
+            var ctx = canvas.getContext("2d");
+            var iter = randomWalk(ctx, 300, 300, 4, "red");
+            setInterval(function() { iter.next(); }, 10);
+        };
+    </script>
+    <style>
+        #mycanvas { border: 1px solid gray;}
+    </style>
+</head>
+<body>
+    <canvas id="mycanvas" width=600 height=600></canvas>
```

Comment:
```suggestion
    <canvas id="mycanvas" width="600" height="600"></canvas>
```
책에서는 그냥 쓰지만 html 내부에는 double quotation으로 묶어주는편이 좋습니다.

Reply:
확인했습니다. 앞으로 double quotation으로 묶겠습니다.